authorization-server/package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "lint": "eslint *.js config db test",
     "start": "node app.js",
-    "test": "mocha test test/db test/config",
+    "test": "OAUTHRECIPES_SURPRESS_TRACE=true mocha ./test/* test/db test/config",
```

Comment:
What's `OAUTHRECIPES_SURPRESS_TRACE=true` do?

Also, why change `test` to `./test/*`? If there's that change, shouldn't there also be a `./test/db` change?